### ISSUE TITLE
Images are never deleted

### ISSUE THREAD

**Issue by Govelius** [Laverna/laverna#154]
I can't read / understand the laverna code, but I can see the Browser's IndexedDB.

Each note has an array of ImageID.
The problem happens when an ImageID gets removed from `notes-db/notes`. The actual data does not get removed from `notes-db/files`

This is not just IndexedDB, the files stay forever in dropbox too.

**Comment by KenrySanchez** [comment]
Is this bug still present??

**Comment by wwebfor** [comment]
Yes, it does. I think it isn't necessarily a bug. More like a feature to add.
